internal/xds/translator/listener.go

```diff
@@ -110,9 +110,6 @@ func (t *Translator) addXdsHTTPFilterChain(xdsListener *listenerv3.Listener, irL
 	}
 
 	if irListener.IsHTTP2 {
```

Comment:
Do we also need to remove ```IsHTTP2``` flag in the irListener? 

Since a listener can have both grpcroutes and httproutes, a listener should not have a protocol property.

Reply:
its still used to attach proto specific filters such as `grpc-web`, which we'd ideally not like to include by default

Reply:
Makes sense.

Maybe rename it to HasHTTP2Route to make the name more precise.